test/e2e/tests/account/snap-account-signatures.spec.ts

```diff
@@ -62,8 +59,22 @@ describe('Snap Account Signatures @no-mmi', function (this: Suite) {
           await experimentalSettings.check_pageIsLoaded();
           await experimentalSettings.toggleRedesignedSignature();
 
-          // Run all 5 signature types
+          // Connect the SSK account
           await new TestDapp(driver).openTestDappPage();
+
+          await driver.findClickableElement({ text: 'Connect', tag: 'button' });
```

Comment:
Not a blocker, but there is a "nit" comment from the other PR that relates to these changes

Reply:
addressed here https://github.com/MetaMask/metamask-extension/pull/29761/commits/fc842e9bfe3729e6d0e8147ab0563b66972582f1